src/main.js

```diff
@@ -132,13 +132,16 @@ export default class Auth {
 			// Add a hidden date property to the returned object.
 			// Used mostly to calculate the expiration date for tokens.
 			Object.defineProperty(data, 'expiresAt', { value: Date.parse(response.headers.get('date')) + 3600 * 1000 });
+
+			// Remove unnecessary property
+			delete data.kind;
 			return data;
 		});
 	}
 
 	/**
-	 * Makes sure the user is signed-in and has up-to-date credentials.
-	 * @throws Will throw if the user is not signed-in.
+	 * Makes sure the user is logged in and has up-to-date credentials.
```

Comment:
Maybe logged in should be signed in everywhere? I long for "sign in" as this verb is used in the API.